test/test.js

```diff
@@ -169,7 +169,7 @@ describe('sendEvent', () => {
   it('should call _gaTransform if this.tid is set', () => {
       beforeWebExt(global);
       const m = new Metrics({id: '@my-addon', version: '1.0.2', uid: '12345', tid: 'UA-49796218-47'});
-      const clientStub = sinon.stub(m, '_sendToClient');
+      // const clientStub = sinon.stub(m, '_sendToClient');
```

Comment:
Wasn't sure if this was right here... Since it works with this code being commented out, maybe we can just delete these stubs.